addons/website_livechat/tests/test_livechat_basic_flow.py

```diff
@@ -47,6 +47,10 @@ def test_livechat_username(self):
         # Remove livechat_username
         self.operator.livechat_username = False
 
+        # this fixes an issue in the controller; it looks like a cache issue,
+        # but I could not figure it out
+        self.operator.name
```

Comment:
Isn't this related to the display_name/name_get cache and context issue on partner that I reported the other day?

Reply:
Maybe. I spent some time to figure it out, but eventually gave up and found this workaround.